core/src/main/scala/org/http4s/internal/package.scala

```diff
@@ -0,0 +1,15 @@
+package org.http4s
+
+import cats.effect.Async
+import cats.implicits._
+import scala.concurrent.ExecutionContext
+
+package object internal {
+  def blocking[F[_], A](fa: F[A], blockingExecutionContext: ExecutionContext)(implicit F: Async[F], ec: ExecutionContext): F[A] =
```

Comment:
This looks a lot like @ChristopherDavenport's [linebacker 0.1](https://github.com/ChristopherDavenport/linebacker/blob/series/0.1.x/linebacker/src/main/scala/io/chrisdavenport/linebacker/Linebacker.scala#L19-L24) and [the not ideal combinator](https://gist.github.com/SystemFw/256205f51bf0135e4c6fd95dee4590fc) in @SystemFw's gist.

Better can be done with bracket.  Better still can be done with rank-N types.  But maybe this is the best we can do now?